lib/js-api/src/api/gs-core-rest-api/docs/BulkAssignHoldResponseEntry.md

```diff
@@ -0,0 +1,7 @@
+# BulkAssignHoldResponseEntry
+
+## Properties
+
+| Name      | Type                                                    | Default value | Description                                        |
+| --------- | ------------------------------------------------------- | ------------- | -------------------------------------------------- |
+| **Entry** | [**BulkAssignHoldResponse**](BulkAssignHoldResponse.md) | **null**      | Response received after bulk assign hold operation |
```

Comment:
could you please remove `**null**` for consistency with other files?

Reply:
removed, thanks!